### DISCUSSION TITLE
Problem with inputRange and sorting (solved)

### DISCUSSION THREAD

**Discussion by marineusde** [Power-Components/livewire-powergrid#432] [BUG]
My Problem was:

```
Column::add()
  ->title('Number')
  ->field('number')
  ->sortable()
  ->searchable()
  ->makeInputRange('0', '10000'),
```

The mistake was, that the method "makeInputRange" dont accept the input range as the first parameter, its the datafield. I insert the real field:

```
Column::add()
  ->title('Number')
  ->field('number')
  ->sortable()
  ->searchable()
  ->makeInputRange('number', '10000'),
```

Its a little bit strange, cause the method field() already got the parameter.

**Comment by marineusde** [comment]
Please close, sorry

**Reply by luanfreitasdev** [reply]
Hi!. If you can share with others the problem and how you solved it. Thanks.
